store/src/sql/event.rs

```diff
@@ -332,63 +226,51 @@ impl SqliteEventStore {
         ))?;
         let id = key.cid().context("Event CID is required")?.to_bytes();
         let multihash = hash.to_bytes();
-        sqlx::query!(
-            "INSERT INTO ceramic_one_event_block (event_cid, idx, root, block_multihash, codec) VALUES ($1, $2, $3, $4, $5) on conflict do nothing;",
-            id,
-            idx,
-            root,
-            multihash,
-            code,
-        )
+        sqlx::query(
+            "INSERT INTO ceramic_one_event_block (event_cid, idx, root, block_multihash, codec) VALUES ($1, $2, $3, $4, $5) on conflict do nothing;")
```

Comment:
should this query be moved into the entities section like the others?

Reply:
Good call. Crated an `EventBlockQuery` struct for it